lib/exports.ts

```diff
@@ -0,0 +1,21 @@
+import { visit } from 'unist-util-visit';
+import mdast from './mdast';
+import { isMDXEsm } from '../processor/utils';
+import { MdxjsEsm } from 'mdast-util-mdx';
+
+const EXPORT_NAME_REGEX = /export\s+(?:const|let|var|function)\s+(\w+)/;
```

Comment:
I think this is fine for now, but in the future we may need to pull this info from the ast.